### PR TITLE
Show warning that cookies are required in the backend

### PR DESCRIPTION
**Pull request by carakas**
## Type

- Non critical bugfix
- Enhancement
- Feature

## Resolves the following issues

fixes https://github.com/forkcms/forkcms/issues/1706

## Pull request description

Fixes the alerts that notify the user that javascript or cookies are disabled

### CODE REVIEW

File: src/Backend/Core/Layout/Templates/header.html.twig [hunk 1]
```diff
@@ -48,3 +48,9 @@
     </li>
   </ul>
 </div>
+
+{% include "/Layout/Templates/messaging.html.twig" %}
+
+<div id="ajaxSpinner" style="position: fixed; top: 10px; right: 10px; display: none;">
```
Comment (StijnVrolijk) [inline src/Backend/Core/Layout/Templates/header.html.twig]: This seems rather dirty, perhaps we could have a frontender have a poke at this?

File: src/Backend/Core/Js/backend.js [hunk 2]
```diff
@@ -12,15 +12,13 @@ var jsBackend =
 
     // init, something like a constructor
     init: function () {
-        // variables
-        var $body = $('body');
```
Comment (StijnVrolijk) [inline src/Backend/Core/Js/backend.js]: was this unused?

File: src/Backend/Core/Layout/Templates/messaging.html.twig [hunk 3]
```diff
@@ -4,7 +4,7 @@
       <p>{{ 'err.JavascriptNotEnabled'|trans|ucfirst }}</p>
     </div>
   </noscript>
-  <div id="noCookies" class="formMessage alert alert-danger alert-main" style="display: none;">
+  <div id="noCookies" class="formMessage alert alert-danger alert-main">
```
Comment (StijnVrolijk) [inline src/Backend/Core/Layout/Templates/messaging.html.twig]: I assume this had `display: none` in a CSS file?

File: src/Backend/Core/Layout/Templates/header.html.twig (hunk 1 above)
Comment (carakas) [inline src/Backend/Core/Layout/Templates/header.html.twig]: maybe, but I just copied it over so it wasn't in 3 places anymore

Comment (StijnVrolijk) [inline src/Backend/Core/Layout/Templates/header.html.twig]: @Katrienvh told me she'll take a look at this

File: src/Backend/Core/Js/backend.js (hunk 2 above)
Comment (carakas) [inline src/Backend/Core/Js/backend.js]: it was indeed

File: src/Backend/Core/Layout/Templates/messaging.html.twig (hunk 3 above)
Comment (carakas) [inline src/Backend/Core/Layout/Templates/messaging.html.twig]: fixed in 6f6e10f

File: src/Backend/Core/Layout/Templates/header.html.twig (hunk 1 above)
Comment (carakas) [inline src/Backend/Core/Layout/Templates/header.html.twig]: fixed in 5499551

**Comment by StijnVrolijk** [comment]
@carakas please fix the merge conflicts

**Comment by carakas** [comment]
@StijnVrolijk fixed

**Pull request merged**